README.md

```diff
@@ -53,19 +74,44 @@ To deploy, create a playbook which looks like this:
           openhpc_packages: []
     ...
 
-Example Inventory
------------------
 
-And an Ansible inventory as this:
+To drain nodes, for example, before scaling down the cluster to 6 nodes:
 
-    [openhpc_login]
-    openhpc-login-0 ansible_host=10.60.253.40 ansible_user=centos
-
-    [openhpc_compute]
-    openhpc-compute-0 ansible_host=10.60.253.33 ansible_user=centos
-
-    [cluster_control:children]
-    openhpc_login
+    ---
+    - hosts: openstack
+      gather_facts: false
+      
+      roles:
```

Comment:
Some commenting on the process used here would be good.

Reply:
Done